setup.py

```diff
@@ -29,12 +29,17 @@ def package_data(pkg, roots):
     ],
     install_requires=[
         'XBlock',
+        'xblock-utils',
+    ],
+    dependency_links=[
+        'lxml',
```

Comment:
dependency_links is meant to be a list of URLs.  This entry won't do anything.  lxml is another of those transitive needs: this XBlock doesn't use lxml.